api/src/main/java/org/pmiops/workbench/api/DataSetController.java

```diff
@@ -609,6 +624,32 @@ private void formatTimestampValues(List<DataSetPreviewValueList> valuePreviewLis
     return ResponseEntity.ok(dataSetResponse);
   }
 
+  @Override
+  public ResponseEntity<DataDictionaryEntry> getDataDictionaryEntry(
+      Long cdrVersionId, String domain, String domainValue) {
+    CdrVersion cdrVersion = cdrVersionDao.findByCdrVersionId(cdrVersionId);
+
+    if (cdrVersion == null) {
+      throw new BadRequestException("Invalid CDR Version");
+    }
+
+    String omopTable = conceptSetDao.DOMAIN_TO_TABLE_NAME.get(Domain.fromValue(domain));
+
```

Comment:
ultranit: visually I think it is more readable to group logical statements together, i.e. dropping the newline between the lookup and error condition checks - right now there's basically just one newline between every statement, which doesn't help the eye.